### ISSUE TITLE
Make Open Library Fully Mobile Responsive

### ISSUE THREAD

**Issue by mekarpeles** [internetarchive/openlibrary#679]
Currently, the homepage is the only page which is fully mobile responsive. We accomplish this within the plugin/openlibary/home.py view by injecting `v2` as an attribute into the page template after it is rendered and then by changing the templates/site.html content based on whether the page has this `v2` attribute to conditionally include a viewport metatag as well as other necessary changes.

This issue concerns (is the parent for) all changes towards mobile responsiveness on Openlibrary.org

### How do we get there?
- [ ] migrate all pages to the new v2. [Find issues to help with this!](https://github.com/internetarchive/openlibrary/issues?q=is%3Aissue+is%3Aopen+v2+label%3A%22mobile+%26+responsive%22)
- [ ] While doing the above, identify components in every single page and make them responsive to mobile devices #1092
- [ ] Throw out all v2 code and make it the default.
- [ ] Redo CSS for all types of pages and throw out redundancy (#1541, 1313)
- [ ] Remove legacy stylesheets.

**Comment by mekarpeles** [comment]
cc: @html5cat

**Comment by html5cat** [comment]
@mekarpeles could you outline the key pages and relevant files that are highest priority? Any relevant tips/tricks appreciated too!

**Comment by vettyvignesh** [comment]
Hi  I would like to help out with this.

**Comment by mekarpeles** [comment]
Hi @vettyvignesh! As a first step, I think we should prioritize the /search page because it's relatively high traffic and there aren't that many "moving parts".

The goal would be (in the mobile view) to take everything that is currently displayed under the **Zoom In** facet sidebar and turn it into a drop down menu which can be triggered by something like a [filter](https://fontawesome.com/icons/filter?style=solid) icon. With the sidebar moved under the dropdown in the mobile view, the books would be able to take up 100% of the widt.

**Comment by vigneshtdev** [comment]
yeah sure, your guidance will be very helpful. :)

**Comment by mekarpeles** [comment]
Happy to help! Ask away. Have you had a chance to get a developer environment setup yet?

If you send me a message to mek@archive.org, we'll be happy to add you to our community slack channel.

**Comment by vigneshtdev** [comment]
I have indeed set up a dev environment. As you pointed out before that first we ought to focus on /serach pages so the current state on an iPhone is:

![localhost_8080_search iphone 5_se](https://user-images.githubusercontent.com/19593072/40276927-6f82f8c8-5c33-11e8-98ec-b4dfce7b9c86.png)

So what are major changes you think should be implemented to make it more responsive?

**Comment by mekarpeles** [comment]
Thanks! If you make a search, you'll notice the search results interface changes.

![current-search-desktop](https://user-images.githubusercontent.com/978325/40277428-c19e8a38-5bd3-11e8-8f1b-57eb6501fbd6.png)

I imagine we want something a bit closer to:
![search-mobile](https://user-images.githubusercontent.com/978325/40277676-edb19832-5bd7-11e8-9121-833a0986ed0c.png)

**Comment by psyked** [comment]
I've started some work towards this on a branch: https://github.com/psyked/openlibrary/tree/679/feature/fully-responsive-design

For the most part, it's a case of replacing any hardcoded element widths with fluid versions or max-width versions, but there are some elements, such as the carousels, which will require a little more effort.

**Comment by jdlrobson** [comment]
Hurraahhhh

**Comment by jdlrobson** [comment]
I'll cut open more specific tasks as there are still likely to be many problems with mobile friendliness!

**Comment by BrittanyBunk** [comment]
Do we have specific #'s on this? I only see #685.

**Comment by BrittanyBunk** [comment]
One of the issues (I don't see it posed, but correct me if I'm wrong) is that the editions table in works is really bad. I needs: 
- better breaks between editions
- simplified boxes: 1) the read button could be smaller, as I feel people do want to listen more than read and the read button's crowding out finding the listen button, 2) I think we could simplify it to just the year for the wording (image is too small to see, the title's the same - so just repeat - waste of space, publisher - not really important enough to show). Let's take out "libraries near you" and say "nearby: WorldCat", "buy: betterworldbooks". I think it is important for people to find a book nearby in a library or buy it on their phone, so that's good. I just think it goes below the 'read/audio' part.

![image](https://user-images.githubusercontent.com/55302685/71317246-f5dbaa00-2432-11ea-973b-7e3f5900f508.png)

**Comment by BrittanyBunk** [comment]
there's more, but we'll start there.
